.taskcluster.yml

```diff
@@ -136,6 +136,36 @@ tasks:
           owner: mcastelluccio@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("check_metrics")}
```

Comment:
I'd put this task at the end of the data pipeline, so we generate new graphs whenever we train the models.

Reply:
Got it. I put it temporarily in this file so I can iron out the taskcluster running with artifacts declaration. I will put it in the data pipeline once it's working properly.